cookbooks/aws-parallelcluster-computefleet/recipes/config/head_node_fleet_status.rb

```diff
@@ -12,6 +12,8 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
+return if node['cluster']['scheduler'] == 'awsbatch'
```

Comment:
Should we avoid calling this recipe instead of checking for `awsbatch` in `computefleet` cookbook which is tightly coupled to slurm?